setup.py

```diff
@@ -28,51 +28,55 @@ def read(*rnames):
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: Implementation :: CPython',
-        'Programming Language :: Python :: Implementation :: PyPy',
-        'Natural Language :: English',
-        'Operating System :: OS Independent',
-        'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope3'],
-    url='http://pypi.python.org/pypi/zope.error',
-	packages=find_packages('src'),
-	package_dir = {'': 'src'},
-    namespace_packages=['zope',],
-    install_requires=['setuptools',
-                      'zope.exceptions',
-                      'zope.interface',
-                      'zope.location',
-                      'persistent', # error.py imports from persistent
-                      'six'
-                      ],
-    extras_require=dict(
-          test=[
+      version='4.4.0.dev0',
+      author='Zope Foundation and Contributors',
+      author_email='zope-dev@zope.org',
+      description="An error reporting utility for Zope3",
+      long_description=(
+          read('README.rst')
+          + '\n\n' +
+          read('CHANGES.rst')
+      ),
+      license='ZPL 2.1',
+      keywords="zope3 error",
+      classifiers=[
+          'Development Status :: 5 - Production/Stable',
+          'Environment :: Web Environment',
+          'Intended Audience :: Developers',
+          'License :: OSI Approved :: Zope Public License',
+          'Programming Language :: Python',
+          'Programming Language :: Python :: 2',
+          'Programming Language :: Python :: 2.7',
+          'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3.4',
+          'Programming Language :: Python :: 3.5',
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: Implementation :: CPython',
+          'Programming Language :: Python :: Implementation :: PyPy',
+          'Natural Language :: English',
+          'Operating System :: OS Independent',
+          'Topic :: Internet :: WWW/HTTP',
+          'Framework :: Zope3',
+      ],
+      url='http://github.com/zopefoundation/zope.error',
+      packages=find_packages('src'),
+      package_dir={'': 'src'},
+      namespace_packages=['zope',],
+      install_requires=[
+          'setuptools',
+          'zope.exceptions',
+          'zope.interface',
+          'zope.location',
+          'persistent', # error.py imports from persistent
+          'six',
+      ],
+      extras_require={
+          'test': [
               'zope.testing >= 3.8',
-              ]),
-    test_suite = 'zope.error.tests',
-    include_package_data = True,
-    zip_safe = False,
-    )
+              'zope.testrunner',
+          ],
+      },
+      test_suite='zope.error.tests',
```

Comment:
On our end, no. But as we recently learned, some downstream projects use `setup.py test` still when packaging, at least until it breaks. If we take the `test_suite` out, then it never breaks---it always returns successfully. That sort of false positive seems worse to me than helping downstream change test runners.
```
$ python setup.py test
running test
running egg_info
writing requirements to src/zope.error.egg-info/requires.txt
writing dependency_links to src/zope.error.egg-info/dependency_links.txt
writing src/zope.error.egg-info/PKG-INFO
writing top-level names to src/zope.error.egg-info/top_level.txt
writing namespace_packages to src/zope.error.egg-info/namespace_packages.txt
reading manifest file 'src/zope.error.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'src/zope.error.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
$ echo $?
0
```